broskichat-app/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import 'bulma/css/bulma.css'
```

Comment:
i thin we'll want the bulma.css import to come before our styles, that way we have the ability to override bulma and make use of CSS's cascading properties